src/maker.js

```diff
@@ -71,6 +72,7 @@ export async function instantiateMaker({
       trezorPlugin,
       ledgerPlugin,
       [walletLinkPlugin, walletLinkPluginConfig],
+      [mewconnectPlugin, walletLinkPluginConfig],
```

Comment:
why the reuse of walletLinkPluginConfig? If the config is necessary, use a different object.